confero/settings.py

```diff
@@ -75,15 +78,15 @@
 
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
-
+db_prefix = 'RDS_DB_' if 'RDS_DB_NAME' in os.environ else 'DB_'
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.getenv("DB_NAME", "confero"),
-        'USER': os.getenv("DB_USER", "postgres"),
-        'PASSWORD': os.getenv("DB_PASSWORD", "postgres"),
-        'HOST': os.getenv("DB_HOST", "localhost"),
-        'PORT': os.getenv("DB_PORT", "5432"),
+        'NAME': os.environ[db_prefix + 'NAME'],
```

Comment:
Nice! How's you get circle working, though? `os.getenv` lets you pass a default value as a second parameter, which let me set the default `DB_NAME` to `confero`, which matched what Circle wanted. I'd think that you'd either have to keep those defaults with `os.environ(db_prefix + 'NAME', "confero")`, or else pass in something like this [here](https://github.com/project-confero/confero/blob/master/.circleci/config.yml#L6):

```bash
      - image: circleci/python:3.7
        environment:
          DB_NAME: confero
      - image: circleci/postgres
        environment:
          POSTGRES_DB: confero
          POSTGRES_USER: postgres
```

Or better yet, both, to make the circle setup more explicit. But it's obviously working already, so don't let this block you from merging if you want to!